docs/unified.rst

```diff
@@ -0,0 +1,25 @@
+.. _gs-unified:
+
+Reading/writing to any format with the unified I/O interface
+============================================================
+
+The regions package provides the functionality to serialise and de-serialise
+Python lists of `~regions.Region` objects in any of the supported formats using
+the unified read/write interface, consisting of the
+:meth:`~regions.ShapeList.read` and  :meth:`~regions.ShapeList.write` methods.
+The following example reads regions from a FITS file and writes them to a DS9
+region file.
+
+.. code-block:: python
+
+    >>> from regions import ShapeList
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> filename = get_pkg_data_filename('data/fits_region.fits',
+    ...                                  package='regions.io.fits.tests')
+    >>> regions = ShapeList.read(filename)
+    >>> print(regions[0])
+    Region: CircleSkyRegion
+    center: <SkyCoord (ICRS): (ra, dec) in deg
+        (211.06231757, 54.49779926)>
+    radius: 0.05220229834087168 deg
```

Comment:
```suggestion
    radius: 0.05220229834 deg
```

Reply:
@larrybradley, I'm going to make these suggested changes manually, because GitHub is giving me some trouble when I try to apply them.

![Screen Shot 2021-04-21 at 12 47 49](https://user-images.githubusercontent.com/728407/115591147-cd222980-a29f-11eb-9e62-94303a289285.png)